README.md

```diff
@@ -1,3 +1,6 @@
+> **Info**: This project is **outdated** and **not maintained**. For an up to date framework
+with the latest and similar teck stack you should bookmark [**gae-init**](https://github.com/gae-init/gae-init).
```

Comment:
@lipis: typo in "teck stack"... should be "tech stack" (or perhaps just use "stack")?